rollup.config.js

```diff
@@ -0,0 +1,32 @@
+import commonjs from '@rollup/plugin-commonjs';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import terser from '@rollup/plugin-terser';
+import typescript from '@rollup/plugin-typescript';
+import replace from '@rollup/plugin-replace';
+
+export default [
+  { file: './src/index.tsx', name: 'leckerli' },
+  { file: './src/gtm.ts', name: 'leckerli-gtm' },
+].map(({ file, name }) => ({
+  input: file,
+  output: {
+    compact: true,
+    sourcemap: false,
+    format: 'iife',
+    dir: './dist/assets',
+    entryFileNames: name + '.min.js',
+    inlineDynamicImports: true,
+  },
+  plugins: [
+    replace({
+      'process.env.NODE_ENV': JSON.stringify('production'),
```

Comment:
We also need to replace `APP_VERSION`.

see https://github.com/antistatique/leckerli/blob/78d0d1b3ad1b314b4a85fdc9e15e31be250e9b82/vite.config.ts#L23 (required for the GTM autoload feature if I remember correctly)

I don't really like to have different config / system for the development and for the production build.

Reply:
Done; I add the missing `APP_VERSION` and I remove Vite from the project to keep only Rollup for the JS, Tailwind for CSS and BrowserSync to serve HTML during dev. Still, I'm not entirely satisfied to loose Vite DX... 🤔